src/scene/skin-instance.js

```diff
@@ -86,8 +86,14 @@ class SkinInstance {
         }
     }
 
-    // resolved skin bones to a hierarchy with the rootBone at its root.
-    // entity parameter specifies the entity used if the bone match is not found in the hierarchy - usually the entity the render component is attached to
+    /**
+     * Resolved skin bones to a hierarchy with the rootBone at its root.
+     *
+     * @param {import('../framework/entity.js').Entity} rootBone - A reference to the entity to be used as
+     * the root bone.
+     * @param {import('../framework/entity.js').Entity} entity - Specifies the entity used if the
+     * bone match is not found in the hierarchy - usually the entity the render component is attached to.
```

Comment:
`SkinInstance#resolve` does not currently appear as public API: https://api.playcanvas.com/classes/Engine.SkinInstance.html
Should this have an @ignore?

Reply:
Thank you for pointing this out, making it public through the comment was unindented, I will add the `@ignore` tag (somehow I always forget it).